client/src/components/CreateCardForm/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Card, CardBody, Input } from 'mdbreact'
+
+const CreateCardForm = props => (
+  <div className="col-sm-3 mx-auto">
+    <Card>
+      <CardBody>
+        <form className="text-left" onSubmit={props.handleSubmit}>
+          <Input
+            value={props.description}
+            onChange={props.handleChange}
+            icon="list prefix grey-text left"
+            label="What to do?"
+          />
+          <div className="col-sm-4 mx-auto">
+            <Input
+              type='submit'
+              value='Add'
+              className="btn btn-deep-orange waves-effect waves-light"
+            />
+          </div>
+        </form>
+      </CardBody>
+    </Card>
+  </div>
+)
+
+CreateCardForm.propTypes = {
+  description: PropTypes.string.isRequired,
```

Comment:
![image](https://user-images.githubusercontent.com/3635774/49079653-409bea00-f252-11e8-91c4-dffc44382f5e.png)

Thoughts?

Reply:
@AleksSenkou I don't have this error. What have you done?

Reply:
@DDKatch right after starting the dev server

Reply:
@AleksSenkou let's do a PP for that, I can't figure out what's wrong.

Reply:
@DDKatch firstly, let's try to figure out by chat. Do you have this error on server start?
@nikitasakov do you?

Reply:
@AleksSenkou No, I don't